projects/plugins/jetpack/modules/carousel/jetpack-carousel.css

```diff
@@ -344,7 +257,7 @@ div.jp-carousel-buttons a:hover {
 
 div.jp-carousel-buttons a.jp-carousel-commentlink,
 a.jp-carousel-image-download {
-	background: url(./images/carousel-sprite.png?5) no-repeat;
+	background: url( ./images/carousel-sprite.png?5 ) no-repeat;
```

Comment:
That's a good point. It may be safer to build the path from a value passed via `wp_localize_script`.

Reply:
It currently works this way on WPcom (https://opengrok.a8c.com/source/xref/trunk/wp-content/mu-plugins/carousel/jetpack-carousel.css?r=227547#346) We haven't changed this part of the code at all, the css has just been formatted to add the spaces.  Checking on WPcom with current carousel it is working with this path with the wp-content path prepended:
<img width="604" alt="Screen Shot 2021-06-30 at 9 38 52 AM" src="https://user-images.githubusercontent.com/3629020/123871308-aeb76900-d987-11eb-8351-809f50ccc39f.png">

Reply:
It could still be worth exploring another approach to this at some point, eg. in the case of WPcom mapping to images on  CDN, so have added an issue for this https://github.com/Automattic/jetpack/issues/20221

Reply:
You're absolutely right that the code was already there, @glendaviesnz, I don't know how I missed that! 🤦 Sorry about that!

Thank you for creating that issue 👍